projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -131,7 +131,7 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
         inView(this.host)
           .pipe(takeUntil(this.destroyed))
           .subscribe(() => {
-            this.createInstance();
+            this.createInstanceOnNonNilValue();
```

Comment:
In your case, the value you pass to tippy is static or dynamic?

Reply:
It may change depending on the domain model, so it's dynamic in some cases.